README.md

```diff
@@ -43,9 +43,9 @@ You can use the pairinteraction software as a Python library, see our [quick sta
 pairinteraction gui
 ```
 
-[about-link]: https://www.pairinteraction.org/pairinteraction/sphinx/html/about.html
-[installation-link]: https://www.pairinteraction.org/pairinteraction/sphinx/html/installation/installation.html
-[tutorial-link]: https://www.pairinteraction.org/pairinteraction/sphinx/html/tutorials/examples_python/quick_start.html
+[about-link]: https://www.pairinteraction.org/pairinteraction/master/about.html
```

Comment:
Shouldn't it be https://www.pairinteraction.org/pairinteraction/latest/about.html, etc.?